src/Symfony/Component/Messenger/Transport/Receiver/QueueReceiverInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Receiver;
+
+use Symfony\Component\Messenger\Envelope;
+
+/**
+ * Some transports may have multiple queues. This interface is used to read from only some queues.
+ *
+ * @author David Buchmann <mail@davidbu.ch>
+ */
+interface QueueReceiverInterface
```

Comment:
Should we make this feature experimental in case we change it in the future? The topic itself shows many possible solutions and we may pick the best one now, but if not it's far easier to go into another direction

Reply:
that sounds like a good idea. @Nyholm do you agree?

how do we mark it experimental, is there a phpdoc tag for it and a standard wording?

Reply:
In case @Nyholm agree, just:
```diff
 * @author David Buchmann <mail@davidbu.ch>
+ * @experimental in 5.3
 */
interface QueueReceiverInterface
```

Reply:
Yes, sure, That will give us 6 more months to decide.